.github/comment.yml

```diff
@@ -13,9 +13,7 @@
       If this PR is targeting `main` branch, then it cannot be merged, as `azure-rest-api-specs-pr` repo `main` branch 
       is mirrored from `azure-rest-api-specs` `main`` branch. <br/>
       If you want to publish the PR to the public repo (`Azure/azure-rest-api-specs`) `main` branch, 
-      please use [OpenAPIHub Publish PR](${openapiHub}/tools/publishpullrequest?pr=${owner}/${repo}/${PRNumber}&to=${to}).
-      </li><li>
-      For further guidance see [Spec Repos](https://eng.ms/docs/products/azure-developer-experience/design/api-specs-pr/api-repos).
+      follow the guidance given in the [spec repos article](https://eng.ms/docs/products/azure-developer-experience/design/api-specs-pr/api-repos).
```

Comment:
@maririos @JonathanCrd I removed mention of OpenAPI Hub here. Does this look right?

Reply:
So we still have this functionality and we still let people use it. Looking at the docs provided, there is no information there about `Push to public` so I would recommend to keep the comment information and once @JonathanCrd looks into this feature and where/how we are going to port it, we can update the guidance here

Reply:
@maririos @JonathanCrd
https://github.com/Azure/azure-rest-api-specs/pull/28006